### DISCUSSION TITLE
can i make window transparent both in normal and fullscreen mode?

### DISCUSSION THREAD

**Discussion by HyperCherry** [hyprwm/Hyprland#2663] [Q&A]
I want to make my VSCode transparent and tried
```yml
windowrulev2 = opacity 0.8 override 0.8 override, class:^(Code)
```
It works well when the window in the normal size.But it doesn't get transparent in full screen mode.is there any way to fix it?
![screenshot_2023-07-07-222136](https://github.com/hyprwm/Hyprland/assets/20416586/cc0577a3-42c3-41fa-8bfc-79c1600d8a7b)
![screenshot_2023-07-07-222142](https://github.com/hyprwm/Hyprland/assets/20416586/d3581264-5200-47e0-9fae-079310137fee)

**Comment by druncopsyken** [comment]
There is a field for it under decorations. try setting fullscreen_opacity = 0.8 within the decoration block. The Wiki is a good reference for all the fields, since they aren't all listed in the default configuration file.

**Reply by HyperCherry** [reply]
I want to make my VSCode transparent only, but when I write this in the configuration, all the windows become transparent
```
decoration {
	fullscreen_opacity = 0.8
}
```
